components/match2/wikis/counterstrike/match_group_input_custom.lua

```diff
@@ -39,7 +39,13 @@ local CustomMatchGroupInput = {}
 ---@param options table?
 ---@return table
 function CustomMatchGroupInput.processMatch(match, options)
-	local finishedInput = match.finished --[[@as string?]]
+	---@type string
+	local finishedInput = tostring(Logic.nilOr(
+		Logic.readBoolOrNil(match.finished),
+		Variables.varDefault('tournament_status'),
+		match.finished,
+		''
+	))
```

Comment:
```suggestion
	local finishedInput = Logic.nilIfEmpty(match.finished) or Variables.varDefault('tournament_status')
```
Shouldn't this work for everything? 

The current suggested implementation would fail in a specific edge case (eg. tournament_status = postponed, finished=skip/cancelled)

Reply:
the difference is what happens on non empty non bool input on finished while also having `tournament_status` wiki var set
personally i would be fine with the suggestion, not sure what CS wiki wants though
maybe @iMarbot can tell us how it should behave with that edge case